src/components/fields/image-upload/image-input/image-input.styles.ts

```diff
@@ -13,21 +17,21 @@ export const Wrapper = styled.div`
 		/* Generated background-image for the dashed border from https://kovart.github.io/dashed-border-generator/  */
 		return `background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='4' ry='4' stroke='${color}' stroke-width='4' stroke-dasharray='8%2c 8' stroke-dashoffset='8' stroke-linecap='square'/%3e%3c/svg%3e");`;
 	}}
```

Comment:
why remove the bottom margin for the wrapper? it is meant to space out this field from the next field.

Reply:
wrapper only has one child ([file](https://github.com/LifeSG/web-frontend-engine/blob/main/src/components/fields/image-upload/image-input/image-input.tsx#L162)) so I thought it was redundant

Reply:
oh this is not for the last child of Wrapper?